capture/src/capture.rs

```diff
@@ -15,14 +15,14 @@ use metrics::counter;
 use time::OffsetDateTime;
 use tracing::instrument;
 
-use crate::billing_limits::QuotaResource;
 use crate::event::{Compression, ProcessingContext};
+use crate::limiters::billing_limits::QuotaResource;
```

Comment:
nit: `limiters::billing_limits` is a bit of a stutter. if we're re-orging, `limiters::billing` makes a bit more sense imo